ElementX/Sources/Screens/HomeScreen/View/HomeScreen.swift

```diff
@@ -20,10 +20,16 @@ import SwiftUI
 import SwiftUIIntrospect
 
 struct HomeScreen: View {
+    @Environment(\.colorScheme) private var colorScheme
     @ObservedObject var context: HomeScreenViewModel.Context
     
     @State private var scrollViewAdapter = ScrollViewAdapter()
     @State private var isSearching = false
+    @State private var bloomView: UIView?
```

Comment:
Maybe useful to group these together now there is so many of them 🙃
```suggestion
    
    // Bloom components
    @State private var bloomView: UIView?
```